css/styles.css

```diff
@@ -0,0 +1,241 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Roboto", Helvetica, Arial, Verdana, sans-serif;
+}
+
+.space-b {
+  justify-content: space-between;
+}
+
+.absolute {
+  position: absolute;
+}
+
+.font-blue {
+  color: #08c;
+  text-decoration: none;
+}
+
+.w-1200px {
+  width: 95vw;
+  min-width: 1200px;
+}
+
+.d-flex {
+  display: flex;
+}
+
+.footer-nav ul .flex-item {
+  border-right: 1px solid rgb(155, 145, 145);
+  justify-content: center;
+  padding-right: 5px;
+}
+
+.main-nav {
+  z-index: 10;
+  top: 75px;
+  right: calc((100% - 900px) / 2);
+  width: 955px;
+  background: linear-gradient(
+    #585858 0%,
+    #747474 4%,
+    #747474 35%,
+    #6e6e6e 50%,
+    #646464 50%,
+    #5c5c5c 65%,
+    #7e7d7d 98%,
+    #747474 98%,
+    #585858 100%
+  );
+  border-radius: 4px;
+}
+
+.main-nav .fab {
+  font-size: 22px;
+}
+
+.main-nav ul {
+  list-style: none;
+  line-height: 25px;
+  align-content: center;
+}
+
+.main-nav li {
+  justify-content: center;
+  border-right: 1px solid rgba(0, 0, 0, 0.15);
+  width: 105px;
+  padding: 4px 0;
+}
+
+.main-nav li a {
+  color: #fff;
+  text-decoration: none;
+  font-size: 12px;
+  font-weight: bold;
+}
+
+.main-nav .search {
+  position: relative;
+}
+
+.main-nav .flex-item:hover {
+  cursor: pointer;
+  background: linear-gradient(
+    #414040 0%,
+    #5a5a5a 4%,
+    #5f5f5f 35%,
+    #5a5a5a 50%,
+    #504f4f 50%,
+    #505050 65%,
+    #5e5d5d 98%,
+    #5e5d5d 98%,
+    #585858 100%
+  );
+}
+
+.main-nav .search .fas {
+  color: white;
+  right: 90px;
+  top: 12px;
+  font-size: 12px;
+}
+
+.main-nav input {
+  width: 95px;
+  height: 20px;
+  border-radius: 10px;
+  background: linear-gradient(#7c7c7c 0%, #606060 50%, #747474 90%);
+  margin: 8px 15px 0 10px;
+}
+
+.main-div {
+  margin: 0 auto;
+  height: 80vh;
+  background: url(https://web.archive.org/web/20140301004302im_/http://images.apple.com/v/home/as/images/your_verse_hero.jpg)
```

Comment:
- [ ] The main background should cover the whole page (no whitespace).

![image](https://user-images.githubusercontent.com/61479648/91079374-581c2880-e61a-11ea-9478-d3fe4370e1aa.png)

| Yours | Original |
|:---:|:---:|
|![image](https://user-images.githubusercontent.com/61479648/91079445-6e29e900-e61a-11ea-904e-8b9a5f96794b.png) |![image](https://user-images.githubusercontent.com/61479648/91079489-7aae4180-e61a-11ea-98f7-9722c5b4310b.png)|